# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdebindings/kdepy.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- kdebindings-3.2.1/dcoppython/lib/pydcop.py.orig	2004-04-09 14:22:19.645982533 +0100
+++ kdebindings-3.2.1/dcoppython/lib/pydcop.py	2004-04-09 14:23:00.486753945 +0100
@@ -32,10 +32,10 @@
 	return pcop.disconnect_dcop_signal(sender,senderObj,signal,receiverObj,slot)
 
 class DCOPApplication(object):
-        def __init__( self, name ):
-                self.name = name
+	def __init__( self, name ):
+		self.name = name
 
-        def __getattr__( self, item ):
+	def __getattr__( self, item ):
 		if item == "__repr__":
 			return object.__repr__
 		if item == "__str__":
@@ -44,15 +44,15 @@
 			return object.__call__
 		if item == "_objects_":
 			return pcop.obj_list(self.name)
-                return DCOPObject( self.name, item )
+		return DCOPObject( self.name, item )
 
 	def _object_(self, object):
 		return DCOPObject( self.name, object )
 
 class DCOPObject(object):
-        def __init__( self, appname, name ):
-                self.appname = appname
-                self.name = name
+	def __init__( self, appname, name ):
+		self.appname = appname
+		self.name = name
 
 	def __repr__( self ):
 		return "DCOPObject(%s,%s)" % ( self.appname, self.name )
@@ -60,7 +60,7 @@
 	def __str__( self ):
 		return "DCOPObject(%s,%s)" % ( self.appname, self.name )
 
-        def __getattr__( self, item ):
+	def __getattr__( self, item ):
 		if item == "__repr__":
 			return object.__repr__
 		if item == "__str__":
@@ -69,16 +69,16 @@
 			return object.__call__
 		if item == "_methods_":
 			return pcop.method_list( self.appname, self.name )
-                return DCOPMethod( self.appname, self.name, item )
+		return DCOPMethod( self.appname, self.name, item )
 
 	def _method_(self, method):
 		return DCOPMethod( self.appname, self.name, method )
 
 class DCOPMethod(object):
-        def __init__( self, appname, objname, name ):
-                self.appname = appname
-                self.objname = objname
-                self.name = name
+	def __init__( self, appname, objname, name ):
+		self.appname = appname
+		self.objname = objname
+		self.name = name
 
 	def __repr__( self ):
 		return "DCOPMethod(%s,%s,%s)" % ( self.appname, self.objname, self.name )
@@ -86,7 +86,7 @@
 	def __str__( self ):
 		return "DCOPMethod(%s,%s,%s)" % ( self.appname, self.objname, self.name )
 
-        def __call__( self, *args ):
+	def __call__( self, *args ):
 		return pcop.dcop_call( self.appname, self.objname, self.name, args )
 
 class DCOPServer(object):
